src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -245,7 +245,7 @@ const TempRHGraph: React.FunctionComponent<Props> = ({
         .range([0, width])
       const yTempScale = d3
         .scaleLinear()
-        .domain([-10, 45])
+        .domain([-10, 40])
```

Comment:
Line 531 also needs to be changed
svg.append('g').call(d3.axisLeft(yTempScale).tickValues([-10, 5, 20, 35, 40]))